zbus_macros/src/iface.rs

```diff
@@ -62,17 +63,21 @@ impl<'a> Property<'a> {
         Self {
             read: false,
             write: false,
+            emits_changed_signal: PropertyEmitsChangedSignal::True,
             ty: None,
             doc_comments: quote!(),
         }
     }
 }
 
+#[derive(PartialEq)]
```

Comment:
This change went into the wrong commit. This should be part of the commit adding `MethodType` enum.